docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -59,23 +59,104 @@ For static onboarding, be sure to use the [Gateway static definition example](#g
+      RACDCERT ID(ZWESVUSR) CONNECT(ID(ZWESVUSR) LABEL('DigiCert CA') RING(ZoweKeyring) USAGE(CERTAUTH))
+      SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
+      ```
+
+    - ACF2
+      
+      ```
+      ACF
+      SET PROFILE(USER) DIV(CERTDATA)
+      INSERT CERTAUTH.SYSYROOT DSNAME('SHARE.SYSY.ROOTCA.CER') LABEL(DigiCert Root CA) TRUST
+      INSERT CERTAUTH.SYSZINTR DSNAME('SHARE.SYSZ.INTERCA.CER') LABEL(DigiCert CA) TRUST
+      F ACF2,REBUILD(USR),CLASS(P),DIVISION(CERTDATA)
+      
+      SET PROFILE(USER) DIVISION(KEYRING)
+      CONNECT CERTDATA(CERTAUTH.SYSYROOT) LABEL(DigiCert Root CA) KEYRING(ZWESVUSR.ZOWERING) USAGE(CERTAUTH)
+      CONNECT CERTDATA(CERTAUTH.SYSZINTR) LABEL(DigiCert CA) KEYRING(ZWESVUSR.ZOWERING) USAGE(CERTAUTH)
+      F ACF2,REBUILD(USR),CLASS(P),DIVISION(KEYRING)
+      ```
+      
+    - TopSecret
+      
+      ```
+      TSS ADD(CERTAUTH) DCDS(SHARE.SYSY.ROOTCA.CER)  DIGICERT(SYSYROOT) LABLCERT('DigiCert Root CA') TRUST
+      TSS ADD(CERTAUTH) DCDS(SHARE.SYSZ.INTERCA.CER)  DIGICERT(SYSZINTR) LABLCERT('DigiCert CA') TRUST
+      TSS ADD(ZWESVUSR) KEYRING(ZOWERING) RINGDATA(CERTAUTH,SYSYROOT) USAGE(CERTAUTH)
+      TSS ADD(ZWESVUSR) KEYRING(ZOWERING) RINGDATA(CERTAUTH,SYSZINTR) USAGE(CERTAUTH)
+      ```
+
+- Import root and intermediate (if applicable) public key certificate of the Cloud Gateway running on system X into the truststore of the domain Gateways running on system Y and Z.
+
+  - **PKCS12**
+
+    Use the following example of the keytool commands:
+
+    `keytool -import -file x/keystore/local_ca/local_ca.cer -alias gateway_x -keystore y/keystore/localhost/localhost.truststore.p12`
+
+    `keytool -import -file x/keystore/local_ca/local_ca.cer -alias gateway_x -keystore z/keystore/localhost/localhost.truststore.p12`
+  
+  - **Keyring**
+
+     Use the following examples of the commands to add certificates from the dataset and connect them to the keyrings used by domain Gateways:
+  
+    - RACF
+  
+      ```
+      RACDCERT ADD('SHARE.SYSX.INTERCA.CER') ID(ZWESVUSR) WITHLABEL('DigiCert CA1') TRUST
+      RACDCERT ID(ZWESVUSR) CONNECT(ID(ZWESVUSR) LABEL('DigiCert CA1') RING(ZoweKeyring) USAGE(CERTAUTH))
+      SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
+      ```
+  
+    - ACF2
+  
+      ```
+      ACF
+      SET PROFILE(USER) DIV(CERTDATA)
+      INSERT CERTAUTH.SYSXINTR DSNAME('SHARE.SYSX.INTERCA.CER') LABEL(DigiCert CA1) TRUST
+      F ACF2,REBUILD(USR),CLASS(P),DIVISION(CERTDATA)
+      
+      SET PROFILE(USER) DIVISION(KEYRING)
+      CONNECT CERTDATA(CERTAUTH.SYSXINTR) LABEL(DigiCert CA1) KEYRING(ZWESVUSR.ZOWERING) USAGE(CERTAUTH)
+      F ACF2,REBUILD(USR),CLASS(P),DIVISION(KEYRING)
+      ```
+  
+    - TopSecret
+  
+      ```
+      TSS ADD(CERTAUTH) DCDS(SHARE.SYSX.INTERCA.CER)  DIGICERT(SYSXINTR) LABLCERT('DigiCert CA1') TRUST
+      TSS ADD(ZWESVUSR) KEYRING(ZOWERING) RINGDATA(CERTAUTH,SYSXINTR) USAGE(CERTAUTH)
+      ```
```

Comment:
Can we add a step to validate that this MultiTenancy configuration was sucessful?

Reply:
Like "verify" command?

Reply:
Yes.